routers/repo/issue.go

```diff
@@ -387,6 +388,33 @@ func RetrieveRepoMilestonesAndAssignees(ctx *context.Context, repo *models.Repos
 	}
 }
 
+func retrieveProjects(ctx *context.Context, repo *models.Repository) {
```

Comment:
Probably should add a shortcircuit if Projects are not enabled.

Reply:
dont know what you mean - can you pleace explain?